src/tile/tile.scss

```diff
@@ -56,6 +53,15 @@
     }
   }
 
+  &.iui-actionable {
```

Comment:
So then thumbnail should still have scale change on hover if tile is non actionable?

Reply:
Moved the thumbnail scaling into actionable hover, focus within, and `.iui-selected`.